frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/local-volume-set-request-data.ts

```diff
@@ -2,11 +2,27 @@ import { apiVersionForModel } from '@console/internal/module/k8s';
 import { LocalVolumeSetModel } from '../../models';
 import { LocalVolumeSetKind, DiskType, DiskMechanicalProperties } from './types';
 import { State } from './state';
-import { LOCAL_STORAGE_NAMESPACE, HOSTNAME_LABEL_KEY, LABEL_OPERATOR } from '../../constants';
+import {
+  LOCAL_STORAGE_NAMESPACE,
+  HOSTNAME_LABEL_KEY,
+  LABEL_OPERATOR,
+  deviceTypeDropdownItems,
+} from '../../constants';
 import { getNodes, getHostNames } from '../../utils';
 
+const getDeviceTypes = (deviceType: string[]) => {
+  const { DISK, PART } = deviceTypeDropdownItems;
+  if ((deviceType.includes(DISK) && deviceType.includes(PART)) || deviceType.length === 0) {
```

Comment:
Since at `deviceType.length === 0` we are taking both `Disk` and `Part`, in multi-select input bar, keep both selected as default, instead of keeping it empty.

Reply:
fixed. now we can pass the pre default values